### PR TITLE
Add proposal iso-hayate.md for a project with Gensim.

### PR DESCRIPTION
**Pull request by isohyt**
Checklist for prospect students for Google Summer of Code 2016:

- [x] I contacted NumFOCUS mentors before writing my proposal
- [x] I showed you my contribution (it can be any form: proof-of-concept project idea, some sample code or just a link to your commits from other project)
- [x] I linked to my sample contribution from the proposal
- [x] I linked to my opened issues in numfocus/gsoc repository from the proposal

### CODE REVIEW

**Comment by tmylk** [comment]
Thanks!
FYI This proposal is for a project with Gensim.

File: 2016/proposals/iso-hayate.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Review gensim word2vec architecture to understand online Word2Vec code in the and resolve AppVeyor error. 
+
+
+
+### July 6th - July 19th
+
+
+
+Evaluating online Word2Vec using [Lee corpus](http://www.socsci.uci.edu/~mdlee/lee_pincombe_welsh_document.PDF) and start writing documentation for gensim users.
```
Comment (tmylk) [inline 2016/proposals/iso-hayate.md]: Please add 2 weeks to write a blog post. Blog post is necessary to attract attention to your contribution. Good blog post solves a practical problem on an interesting corpus.

File: 2016/proposals/iso-hayate.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+Review gensim word2vec architecture to understand online Word2Vec code in the and resolve AppVeyor error. 
+
+
+
+### July 6th - July 19th
+
+
+
```
Comment (tmylk) [inline 2016/proposals/iso-hayate.md]: Please describe HOW will you know if your implementation is correct. What will be your benchmark? What experiment will you run? One full week is needed for an experiment.

**Comment by isohyt** [comment]
Thank you for your helpful comments, @tmylk.
I will fix the parts you instructed me to.

**Comment by isohyt** [comment]
I fixed some parts you instructed me to.
However, I don't figure out the best evaluating method for Word2Vec, so I wrote surveying period in my draft proposal for now.
Please tell me about which methods and datasets are the best for evaluating online Word2Vec.

**Comment by isohyt** [comment]
I've finished writing my proposal and submitted here (https://summerofcode.withgoogle.com/serve/4681206897573888/).

Thank you for your helpful and kind support, @tmylk.

**Pull request merged**